docs/reference/module-api/module-overview.md

```diff
@@ -315,7 +464,7 @@ The following example uses a hypothetical batch operation that might reset piece
 
 ```javascript
 // modules/article/index.js
-module.export = {
+module.exports = {
   batchOperations: {
     add: {
       // This uses a hypothetical`reset` route added in `apiRoutes`
```

Comment:
space

Reply:
I'm not sure what correction you are expecting here.